tasks/loop_foreach_tasks_for_current_app.yml

```diff
@@ -0,0 +1,8 @@
+---
+# This task loop to all tasks in {{ tasks_to_launch }}
+# and launch it.
+
+- include_tasks: "{{ tsk_to_launch }}.yml"
+  loop: "{{ tasks_to_launch }}"
+  loop_control:
+    loop_var: tsk_to_launch
```

Comment:
Following new naming schema, this would be simpler, no?

```yaml
- include_tasks: "{{ task }}.yml"
  loop: "{{ tasks }}"
  loop_control:
    loop_var: task
```

Reply:
done